src/Symfony/Component/HttpFoundation/File/MimeType/MimeTypeExtensionGuesser.php

```diff
@@ -600,6 +600,7 @@ class MimeTypeExtensionGuesser implements ExtensionGuesserInterface
         'application/x-xpinstall' => 'xpi',
         'application/x-xz' => 'xz',
         'application/x-zmachine' => 'z1',
+        'application/x-zip-compressed' => 'zip',
```

Comment:
This list seems to have alphabetical order, could you respect that?

Reply:
oh ffs, would you believe I'm a native speaker 🤦‍♂️

Reply:
Corrected!